generation/serve.py

```diff
@@ -85,6 +85,7 @@ async def generate_model(
 @app.post("/generate_video/")
 async def generate_video(
     prompt: str = Form(),
+    video_res: int = 1088,
```

Comment:
Have you test it? :)
Did you try to pass something else?

Reply:
I think you need to use Form() for this.

Reply:
this way it was easier to ask the question: it is generating 1088 x 1088 video, but I did not figure out how to pass extra parameter via curl to manually change the resolution

Reply:
it is 1088 x 1088 because otherwise the video converter will make a warning and convert the output to this weird size anyway